plugins/inputs/mesos/mesos.go

```diff
@@ -34,34 +36,109 @@ type Mesos struct {
 	SlaveCols  []string `toml:"slave_collections"`
 	//SlaveTasks bool
 	tls.ClientConfig
+	dcosutil.DCOSConfig
 
 	initialized bool
 	client      *http.Client
 	masterURLs  []*url.URL
 	slaveURLs   []*url.URL
 }
 
+// TaggedField allows us to create a predictable hash from each unique
+// combination of tags
+type TaggedField struct {
+	FrameworkName string
+	CallType      string
+	EventType     string
+	OperationType string
+	TaskState     string
+	RoleName      string
+	FieldName     string
+	Resource      string
+	Value         interface{}
+}
+
+func (tf TaggedField) hash() string {
```

Comment:
Can you pull in the recent changes in dcos-telegraf? (bug fix for this hashing function)

Reply:
@gracedo I added #40, PTAL.